src/store.js

```diff
@@ -72,6 +76,16 @@ const getProductsThunk = ()=>{
   }
 }
 
+const addToCart = (item) => {
```

Comment:
Sorry to be picky, can we name this 'addToCartThunk'? Just to be consistent with the rest of the thunks. Can then change the action creator '_addToCart' to either 'addToCart' or 'addCartItem' just to be consistent. But then don't forget to change the export on line 91 as well.

Reply:
Ok, I appreciate your effort.